src/fortanix/Logo/Logo.tsx

```diff
@@ -28,12 +28,14 @@ export const Logo = ({ subtitle, subtitleTrademark, ...propsRest }: LogoProps) =
       )}
     >
       <div className="_logo">
-        <img alt={`Fortanix ${subtitle}`} src={logoImage} className="_icon"/>
+        <img alt={`Fortanix ${subtitle ?? ''}`} src={logoImage} className="_icon"/>
```

Comment:
Maybe add a `.trim()` at the end here to remove the trailing space in case of no subtitle.